tests/test_genie_dae.py

```diff
@@ -112,6 +112,16 @@
     </DBL>
 </Cluster>"""
 
+UPDATE_SETTINGS_XML = """<Cluster>
+    <Name>DAE Updates</Name>
+    <NumElts>3</NumElts>
+    <U32>
+        <Name> Frequency</Name>
+        <Val>5000</Val>
+    </U32>
+</Cluster>
+"""
+
```

Comment:
You should use the full XML herer as per https://github.com/ISISComputingGroup/EPICS-isisdae/blob/master/isisdaeApp/src/xml/update_settings_cluster.xml